### PR TITLE
TRUST QA-21: minimum voting delay

### PR DESCRIPTION
**Pull request by julianmrodri**
* Enforce Minimum voting delay of 1 day (dependes on Network Config)
* Adds test

### CODE REVIEW

File: contracts/plugins/governance/Governance.sol [hunk 1]
```diff
@@ -44,7 +47,12 @@ contract Governance is
         GovernorVotes(IVotes(address(token_)))
         GovernorVotesQuorumFraction(quorumPercent)
         GovernorTimelockControl(timelock_)
-    {}
+    {
+        MIN_VOTING_DELAY =
+            (86400 + NetworkConfigLib.blocktime() - 1) /
```
Comment (tbrent) [inline contracts/plugins/governance/Governance.sol:52]: Let's declare the 86400 figure as a constant just outside the contract

File: contracts/plugins/governance/Governance.sol [hunk 2]
```diff
@@ -30,7 +31,9 @@ contract Governance is
     // 100%
     uint256 public constant ONE_HUNDRED_PERCENT = 1e8; // {micro %}
 
-    // solhint-disable no-empty-blocks
+    // solhint-disable-next-line var-name-mixedcase
+    uint256 public immutable MIN_VOTING_DELAY; // in blocks, based on network
```
Comment (tbrent) [inline contracts/plugins/governance/Governance.sol:35]: let's reference here in the comment that it should be equal to `ONE_DAY`, or whatever we end up calling the 86400 figure as a constant

**Review by tbrent — COMMENTED** [review]
Tests look good

File: contracts/plugins/governance/Governance.sol (hunk 2 above)
Comment (julianmrodri) [reply contracts/plugins/governance/Governance.sol:35]: will do the changes and push

Comment (tbrent) [reply contracts/plugins/governance/Governance.sol:35]: also can use our unit notation `{blocknumber}` here

**Pull request merged**